### ISSUE TITLE
Multiline comments syntax highlight.

### ISSUE THREAD

**Issue by yglukhov** [pragmagic/vscode-nim#11]
Basically subj. Currently multiline comments and doc commnets are not highlighted properly:
```nim
#[
This is a comment.
#[
This comment is nested
]#
]#

##[
Doc comment

##[
Nested doc commnet
]##

]##
```

Unfortunaly github doesnt support it as well =)

**Comment by oderwat** [comment]
I think I fixed that (your example works) with #17 (I also converted the format of the syntax file to make it more readable for me)

**Comment by kosz78** [comment]
Available in 0.5.4 release
